### DISCUSSION TITLE
Training a look but still able to add variety using prompts

### DISCUSSION THREAD

**Discussion by shyt47** [TheLastBen/fast-stable-diffusion#1876] [General]
"I've had some success training faces to achieve a desired look in the past by using more than the recommended 10 pictures, usually around 25-30 images of the same person. However, this approach hasn't been yielding great results lately. So, I tried the recommendations from Ben, which involves using only 10 images with default settings in Colab and then resuming at 200 steps until the desired look is achieved. While I was able to get decent results with this method, I found it challenging to change the appearance of the person in the generated images. Regardless of what I typed in the prompt, the person tended to be in a very similar pose and wear almost the same thing. I'm wondering if anyone has better settings or techniques to achieve more accurate and manipulable results."

**Comment by TheLastBen** [comment]
when you resume the training do not resume the text encoder, meaning after you train the text encoder once, after that set it to 0 then resume only the unet

**Reply by shyt47** [reply]
Yes. I have been doing that, but are the settings I describe otherwise still the best for a face?

**Reply by TheLastBen** [reply]
yes, make sure you have a good dataset
